src/Tokenizer/Tokens.php

```diff
@@ -293,9 +293,6 @@ public function offsetUnset($index): void
      */
     public function offsetSet($index, $newval): void
     {
-        $this->blockStartCache = [];
-        $this->blockEndCache = [];
```

Comment:
i want to run it conditionally and compare performance, let's keep the PR open for now as reminder for myself